crates/cli/Cargo.toml

```diff
@@ -17,6 +18,7 @@ serde_json = { workspace = true }
 serde_yaml = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
+libpq = { workspace = true }
```

Comment:
This may cause issues with linking such that it will run on any machine without external deps.

Please make sure it is statically linked.

Reply:
We no longer use libpq 🎉